packages/components/src/components/numpad/number-grid.tsx

```diff
@@ -1,8 +1,7 @@
-import PropTypes from 'prop-types';
 import React from 'react';
-import NumberButton from './number-button.jsx';
+import NumberButton, { TNumberButton } from './number-button';
 
-const NumberGrid = ({ onSelect }) => {
+const NumberGrid = ({ onSelect }: TNumberButton) => {
```

Comment:
As per [number-button.tsx](https://github.com/niloofar-deriv/deriv-app/pull/89/files#diff-2d635e42c814775f50f938c45c506e6cf2c0b40e8cd4ded7c8a4aa572f8bbef0), `onSelect` & `number` props are mandatory. But here number is not passed. Can u verify this again?